### PR TITLE
PAE 273 - Update register message

### PR DESCRIPTION
**Pull request by luismorenolopera**
Description
---------------
Update new account message

### CODE REVIEW

**Review by diegomillan — CHANGES_REQUESTED** [review]
We should not remove gettext(), also I would recommend to use something like:

`<%- edx.StringUtils.interpolate(gettext('Already have an {platformName} account?'), {platformName: context.platformName }) %>`

The idea is to avoid hardcoded strings whenever possible.

See line 11 on this same file for reference
@luismorenolopera

File: edx-platform/pearson-learninghub-theme/lms/templates/student_account/register.underscore [hunk 1]
```diff
@@ -46,7 +46,7 @@
             </div>
             <div class="section-title lines">
                 <h3>
-                    <span class="text"><%- gettext("or create a new one here") %></span>
+                    <span class="text">Create a new Learning Hub account here</span>
```
Comment (Squirrel18) [inline edx-platform/pearson-learninghub-theme/lms/templates/student_account/register.underscore]: I agree with @diegomillan We should interpolate the platform name and mark the text for extraction. @luismorenolopera

**Review by Squirrel18 — CHANGES_REQUESTED** [review]

File: edx-platform/pearson-learninghub-theme/conf/locale/en/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -3482,3 +3482,6 @@ msgstr ""
 msgid "Create Account"
 msgstr ""
 
+#: edx-platform/pearson-pols-theme/lms/templates/student_account/register.underscore:46
+msgid "Create a new account here"
```
Comment (diegomillan) [inline edx-platform/pearson-learninghub-theme/conf/locale/en/LC_MESSAGES/django.po]: What about the interpolated string? @luismorenolopera ...We should have something like [this](https://github.com/proversity-org/proversity-openedx-themes/blob/613663e9c0637eda722bda8d33006b98f013e9d3/edx-platform/pearson-learninghub-theme/conf/locale/en/LC_MESSAGES/django.po#L3445)   

Can you test a dummy translation just to see if it works?

If you are not planning to extract the string now then ignore my comment @luismorenolopera @Squirrel18

**Review by diegomillan — CHANGES_REQUESTED** [review]

File: edx-platform/pearson-learninghub-theme/conf/locale/en/LC_MESSAGES/django.po (hunk 2 above)
Comment (Squirrel18) [reply edx-platform/pearson-learninghub-theme/conf/locale/en/LC_MESSAGES/django.po]: It's not necessary to extract the translation in this PR, but we must check  the interpolation of the string and make sure it's working as expected.

**Review by Squirrel18 — APPROVED** [review]

**Pull request merged**